topics/about_arrays.js

```diff
@@ -19,8 +19,10 @@ test("length", function() {
 test("splice", function() {
     var daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
     var workingWeek = daysOfWeek.splice(__, __);
-    ok(workingWeek.equalTo([__]), 'what is the value of workingWeek?');
-    ok(daysOfWeek.equalTo([__]), 'what is the value of daysOfWeek?');
+    var weekend = daysOfWeek;
+
+    deepEqual(workingWeek, ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'], 'what is the value of workingWeek?');
+    deepEqual(weekend, ['Saturday', 'Sunday'], 'what is the value of daysOfWeek?');
```

Comment:
Damn. I removed my previous comment here by mistake. 

The thing is - I changed the variable names to indicate which days go where. If you pass 0 and 5 as splice call args then it will pass the test. I checked it.